Jenkinsfile

```diff
@@ -29,225 +23,154 @@ pipeline {
+                failure {
+                    colourText("warn","Stage: ${env.STAGE} failed!")
                 }
-                // stash name: "zip"
             }
         }
 
-        stage("Releases"){
+        stage('Package'){
             agent any
-            when {
-                anyOf {
-                    branch BRANCH_DEV
-                    branch BRANCH_TEST
-                    branch BRANCH_PROD
-                }
+            when{ expression{ isBranch("master") }}
+            environment{ 
+                STAGE = "Package" 
+                DEPLOY_TO = "dev"    
             }
             steps {
-                script {
-                    env.NODE_STAGE = "Releases"
-                    currentTag = getLatestGitTag()
-                    colourText("info", "Found latest tag: ${currentTag}")
-                    newTag =  IncrementTag( currentTag, RELEASE_TYPE )
-                    colourText("info", "Generated new tag: ${newTag}")
-                    //push(newTag, currentTag)
+                dir('gitlab') {
+                    git(url: "$GITLAB_URL/BusinessIndex/${MODULE_NAME}.git", credentialsId: GITLAB_CREDS, branch: "master")
                 }
+                sh 'sbt universal:packageBin'
+                sh "cp target/universal/${ORGANIZATION}-${MODULE_NAME}-*.zip ${DEPLOY_TO}-${ORGANIZATION}-${MODULE_NAME}.zip"
             }
-        }
-
-        stage ('Package and Push Artifact') {
-            agent any
-            when {
-                branch BRANCH_PROD
-            }
-            steps {
-                script {
-                    env.NODE_STAGE = "Package and Push Artifact"
+            post {
+                success {
+                    colourText("info","Stage: ${env.STAGE} successful!")
+                }
+                failure {
+                    colourText("warn","Stage: ${env.STAGE} failed!")
                 }
-                sh """
-                    $SBT clean compile package
-                    $SBT clean compile assembly
-                """
-                colourText("success", 'Package.')
             }
         }
 
-        stage('Deploy'){
+        stage('Deploy CF'){
             agent any
-            when {
-                anyOf {
-                    branch BRANCH_DEV
-                    branch BRANCH_TEST
-                    branch BRANCH_PROD
-                }
+            when{ expression{ isBranch("master") }}
+            environment{ 
+                STAGE = "Deploy CF"
+                DEPLOY_TO = "dev" 
             }
             steps {
-                script {
-                    env.NODE_STAGE = "Deploy"
-                }
                 milestone(1)
-                lock('Deployment Initiated') {
-                    colourText("info", 'deployment in progress')
+                lock('Business Index API Deployment Initiated') {
```

Comment:
We should pick a naming scheme for locks so it's clear a resource is being locked (it's not a description per-se).

It might make sense here to use the CF route as the lock id.